akka-stream-testkit/src/test/scala/akka/stream/testkit/StreamTestKitSpec.scala

```diff
@@ -0,0 +1,125 @@
+/**
+ * Copyright (C) 2015 Typesafe Inc. <http://www.typesafe.com>
+ */
+package akka.stream.testkit
+
+import akka.stream._
+import akka.stream.scaladsl.Source
+import akka.stream.testkit.scaladsl.TestSink
+
+import scala.concurrent.duration._
+
+class StreamTestKitSpec extends AkkaSpec {
+
+  implicit val mat = ActorMaterializer()
+
+  val ex = new Exception("Boom!")
+
+  "A TestSink Probe" must {
+    "#toStrict" in {
+      Source(1 to 4).runWith(TestSink.probe)
+        .toStrict(300.millis) should ===(List(1, 2, 3, 4))
```

Comment:
There is still no reason to use such short timeout. The test will not complete faster.